c2corg_api/views/document_protect.py

```diff
@@ -47,7 +48,18 @@ def post(self):
         """
         document_id = self.request.validated['document_id']
         document = _get_document(document_id)
+
+        # Do nothing if document is already protected.
```

Comment:
Is there a reason why we duplicate the code (DocumentProtectRest&DocumentUnprotectRest). Couldn't we call the same function, with extra params ?

Reply:
You're right. This code could be factorized. I prefer not doing it in this PR since the fix is not really ordered by the association (as far as I know), so I prefer not spending more time on it :)